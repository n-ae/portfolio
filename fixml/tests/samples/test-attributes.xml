<root>
  <item id="1">First</item>
  <item id="1">First</item>
  <item id="2">Second</item>
  <container type="main">
    <data>Content</data>
  </container>
  <container type="backup">
    <data>Content</data>
  </container>
</root>