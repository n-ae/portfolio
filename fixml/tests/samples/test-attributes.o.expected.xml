<root>
  <item id="1">First</item>
  <item id="2">Second</item>
  <container type="main">
    <data>Content</data>
  </container>
  <container type="backup">
  </container>
</root>
